entrypoint.sh

```diff
@@ -33,22 +33,27 @@ if [ "${target}" = "pi1" ]; then
   memory=256m
   root=/dev/sda2
   nic="--net nic --net user,hostfwd=tcp::5022-:22"
+  for fwd in $HOSTFWD; do nic="$nic,hostfwd=$fwd"; done
 elif [ "${target}" = "pi2" ]; then
   emulator=qemu-system-arm
   machine=raspi2b
   memory=1024m
   kernel_pattern=kernel7.img
   dtb_pattern=bcm2709-rpi-2-b.dtb
   append="dwc_otg.fiq_fsm_enable=0"
-  nic="-netdev user,id=net0,hostfwd=tcp::5022-:22 -device usb-net,netdev=net0"
+  nic="-netdev user,id=net0,hostfwd=tcp::5022-:22"
+  for fwd in $HOSTFWD; do nic="$nic,hostfwd=$fwd"; done
+  nic="$nic -device usb-net,netdev=net0"
 elif [ "${target}" = "pi3" ]; then
   emulator=qemu-system-aarch64
   machine=raspi3b
   memory=1024m
   kernel_pattern=kernel8.img
   dtb_pattern=bcm2710-rpi-3-b-plus.dtb
   append="dwc_otg.fiq_fsm_enable=0"
-  nic="-netdev user,id=net0,hostfwd=tcp::5022-:22 -device usb-net,netdev=net0"
+  nic="-netdev user,id=net0,hostfwd=tcp::5022-:22"
+  for fwd in $HOSTFWD; do nic="$nic,hostfwd=$fwd"; done
```

Comment:
Instead of repeating this inside each conditional can we just do it once outside?

Reply:
Yeah definitely